components/cache/cache_pools.rst

```diff
@@ -278,4 +278,13 @@ when all items are successfully deleted)::
     // ...
     $cacheIsEmpty = $cache->clear();
 
+.. tip::
+
+    If the Cache component is used inside a Symfony application, you can remove
+    all the items of a given cache pool with the following command:
+
+    .. code-block:: terminal
+
+        $ ./bin/console cache:pool:clear <cache-pool-name>
```

Comment:
We can use comments to document other usages (IIRC, we do it in other places as well):

```rst
# clears the "cache.app" pool
$ php bin/console cache:pool:clear cache.app

# clears the "cache.validation" and "cache.app" pool
$ php bin/console cache:pool:clear cache.validation cache.app
```